zcash_primitives/src/transaction/mod.rs

```diff
@@ -90,6 +101,7 @@ pub enum TxVersion {
     Sprout(u32),
     Overwinter,
     Sapling,
+    ZcashTxV5,
```

Comment:
This PR adds v5 to `TxVersion`, but it's not intended to actually implement the v5 transaction, correct?

Reply:
Correct.